clang/lib/CIR/CodeGen/CIRGenFunctionInfo.h

```diff
@@ -26,7 +26,6 @@ namespace clang::CIRGen {
 
 struct CIRGenFunctionInfoArgInfo {
```

Comment:
Does this class still have a purpose?

Reply:
Not really. I wanted to leave it in this PR to keep things focused on what I was removing. I do intend to make another change that removes this but I prefer to keep that change separate.